smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/PackageManifestGenerator.kt

```diff
@@ -6,12 +6,15 @@ package software.amazon.smithy.swift.codegen
 
```

Comment:
Q: Just a general Q on manifest generator. AFAIK we don't generate Package.swift for smithy-swift, and this is just used downstream in aws-sdk-swift when we generate Package.swift files for services - am I correct in my understanding there?

Reply:
You are correct.  The only packages that use the PackageManifestGenerator are service clients.  Runtime modules in both aws-sdk-swift and in smithy-swift will have hand-maintained manifests.